packages/x-components/src/x-modules/history-queries/wiring.ts

```diff
@@ -151,5 +151,8 @@ export const historyQueriesWiring = createWiring({
   },
   UserClickedConfirmDisableHistoryQueries: {
     setHistoryQueriesDisabled
+  },
+  UserClickedCloseX: {
+    clearHistoryQueriesQuery
```

Comment:
Shouldn't we add the same wires to the `UserClickedOutOfXModal` `XEvent`?

```suggestion
  UserClickedCloseX: {
    clearHistoryQueriesQuery
  },
  UserClickedOutOfXModal: {
    clearHistoryQueriesQuery
```

Reply:
Luis, fuck, you are right. Sorry and thanks for realising about it.

Reply:
Yes, when we move the modals to its own XModule, we have to rethink all these events